README.md

```diff
@@ -71,7 +71,7 @@ See `Ueberauth.Strategy` for more information on constructing the Ueberauth.Auth
 def application do
   # Add the application to your list of applications.
   # This will ensure that it will be included in a release.
-  [applications: [:logger, :ueberauth]]
+  [applications: [:logger, :oauth2, :ueberauth]]
```

Comment:
@doomspork Ah. That makes sense. So it should be, in the case of the google strategy,

```elixir
[applications: [:logger, :ueberauth_google]]
````

Is that correct?